File: web/catraca/classes/model/Catraca.php
<?php

class Catraca{
	
	private $id;
	private $nome;
	private $ip;
	
	public function setId($id){
		$this->id = intval($id);
	}
	public function getId(){
		return $this->id;
	}
	
	public function setNome($nome){
		$this->nome = $nome;
	}
	public function getNome(){
		return $this->nome;
	}
	
	public function setIp($ip){
		$this->ip = $ip;
	}
	public function getIp(){
		return $this->ip;
	}
	
}


?>

File: java/catraca_virtual/catraca_virtual/src/main/java/br/edu/unilab/unicafe/model/Catraca.java
package br.edu.unilab.unicafe.model;

import java.net.InetAddress;
import java.net.UnknownHostException;

public class Catraca {
	private int id;
	private String nome;
	private boolean financeiroAtivo;
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public String getNome() {
		return nome;
	}
	public void setNome(String nome) {
		this.nome = nome;
	}
	public boolean isFinanceiroAtivo() {
		return financeiroAtivo;
	}
	public void setFinanceiroAtivo(boolean financeiroAtivo) {
		this.financeiroAtivo = financeiroAtivo;
	}
	
	public void maquinaLocal() throws UnknownHostException{
		InetAddress ia = null;
		ia = InetAddress.getLocalHost();
		this.nome = ia.getHostName().toString();
	}
	

}
